src/pattern-library/components/patterns/data/DataTablePage.tsx

```diff
@@ -211,6 +211,20 @@ export default function DataTablePage() {
 ];`}
             />
           </Library.Example>
+          <Library.Example title="Grid DataTables with scroll">
+            <Library.Demo title="Grid DataTables with scroll" withSource>
+              <div className="w-full h-[250px]">
+                <Scroll>
+                  <DataTable
+                    title="A subset of Nabokov's novels with publish date and original language"
+                    rows={nabokovRows}
+                    columns={nabokovColumns}
+                    grid
+                  />
+                </Scroll>
+              </div>
+            </Library.Demo>
+          </Library.Example>
```

Comment:
This example was used to test the usage of a grid table inside a scrollable container.

It highlighted an issue with the use of `border-collapse` with sticky headers (which was in fact documented already inside `Table`), so I had to set borders individually/conditionally in cells to mimic border-collapse, without breaking sticky headers when using the grid.